README.md

```
@@ -3,6 +3,7 @@
 ## Prerequisites
 
 - Both `node.js` and `npm` should be installed globally. You can find the instructions https://nodejs.org/en/download/
```

Markdownlint

Bare URL used

(MD034, no-bare-urls)